scripts/main.js

```diff
@@ -1,14 +1,61 @@
-document.querySelector('#search form')
-  .addEventListener('submit', (event) => {
-    event.preventDefault();
-    const term = event.target.search.value;
-    fetch(`https://emagi-server-8-0.herokuapp.com/search/${term}`)
-      .then((response) => response.json())
-      .then((emojis) => {
-        console.log(emojis)
-        const result = emojis.map((emoji) => emoji.symbol).join('')
-        const resultArea = document.querySelector('#search aside p')
-        resultArea.textContent = result;
-        document.querySelector('#search aside').classList.add('success')
-      })
-  })
\ No newline at end of file
+document.querySelector("#search form").addEventListener("submit", (event) => {
+  event.preventDefault();
+  const term = event.target.search.value;
+  console.log(event.target, "bbbbb");
+  fetch(`https://emagi-server-8-0.herokuapp.com/search/${term}`)
+    .then((response) => response.json())
+    .then((emojis) => {
+      console.log(emojis);
+      const result = emojis.map((emoji) => emoji.symbol).join("");
+      const resultArea = document.querySelector("#search aside p");
+      resultArea.textContent = result;
+      document.querySelector("#search aside").classList.add("success");
+    });
+  event.target.reset();
+});
+
+document.querySelector("#category form").addEventListener("submit", (event) => {
+  event.preventDefault();
+  const category = event.target.category_select.value;
+  fetch(`https://emagi-server-8-0.herokuapp.com/categories/${category}`)
+    .then((response) => {
+      response.json().then((emojis) => {
+        const result = emojis.map((emoji) => emoji.symbol).join("");
+        const resultAreas = document.querySelector("#category aside p");
+        resultAreas.textContent = result;
+        document.querySelector("#category aside").classList.add("success");
+      });
+    })
+    .catch(console.log);
+});
```

Comment:
The directions specify that you should get one emoji matching the category. Check out the random function in the random file and see how you can incorporate this function to only return on emoji.